### ISSUE TITLE
3rd person glitch after battle makes "remember to exorcise" almost impossible to complete

### ISSUE THREAD

**Issue by Noqxious** [ObliqueNET/Terra#271]
Remember to Exorcise - u can complete this one but its still broken in a very annoying way, after the boss fight against the possessed guy ur supposed to be dragged into a fight vs the dusknoir possessing him, but u get 3rd person glitched consistently every time u beat him (where u remain in ur battle view but in third person at the same time, when u look around ur character looks around too but cant move, can be fixd by relogging). this wouldnt be an issue as u can normally just relog; but the quest resets u to before the boss fight if u log off or leave the cave, so u cant actually finish the quest without using a workaround to fix the issue. the easiest ways to workaround it are to either stand behind the boss in the exact spot where the dusknoir will spawn, so that when ur third person glitched u can still send out ur poke (it falls at ur feet rather than shooting where u aim when ur in this glitch) and it triggers the fight with the boss, or to fight it from wherever then get someone to stand inside the dusknoir, and tp to them whilst glitched, then throw out ur poke to start the fight

TL:DR - beating boss glitches u 100% of the time, wouldnt be an issue except if u tp away/relog to fix it it resets ur quest progress, and u need to be able to move to fight the dusknoir

**Comment by BlitzAsogi** [comment]
Reiterating this issue, went to test this quest today and getting the exact same issue. I was only able to complete it by setting myself to the stage vs the dusknoir

**Comment by Vohras2** [comment]
Could this be the same issue as what we had with Trainer's Way of Life?

**Comment by BlitzAsogi** [comment]
Owski has reuploaded the quest to be completable even if the glitch happens. I've also personally enlarged the roof of the cave to try reduce the chance of it happening.
